src/ports/postgres/modules/graph/test/wcc.sql_in

```diff
@@ -276,3 +276,12 @@ SELECT graph_wcc_num_cpts(
 SELECT assert(relative_error(num_components, 3) < 0.00001,
         'Weakly Connected Components: Incorrect largest component value.'
     ) FROM count_table WHERE user_id1=1;
+
+DROP TABLE IF EXISTS wcc_warm_start_out, wcc_warm_start_out_summary;
+SELECT weakly_connected_components('v2',NULL,'e2',NULL,'wcc_warm_start_out', 'user_id', 2);
```

Comment:
1. That will error as expected since the message table do not exist for them.
2. Added
3. Fixed
4. Added a check to make sure the same set of arguments are passed for warm start (except num of iterations)